src/main/java/org/launchcode/water_garden_tour/models/User.java

```diff
@@ -29,7 +29,6 @@
     @Email(message="Email is not valid.")
     private String username;
 
-    @NotBlank(message="Password is required.")
```

Comment:
Removed because we determined the RegisterDTO was doing this validation, this change allowed validation to work on users/update without passing password through.